MonoDevelop.FSharpBinding/FSharpSymbolHelper.fs

```diff
@@ -398,18 +398,33 @@ module SymbolTooltips =
             if signatureOnly then retType
             else asType Keyword modifiers ++ functionName ++ asType Symbol "() :" ++ retType 
         | many ->
-                let allParams =
-                  many
-                  |> List.map(fun listOfParams ->
-                                  listOfParams
-                                  |> List.map(fun p -> formatName indent padLength p ++ asType UserType (escapeText (p.Type.Format displayContext)))
-                                  |> String.concat (asType Symbol " *" ++ "\n"))
-                  |> String.concat (asType Symbol " ->" + "\n") 
-
-                let typeArguments =
-                    allParams +  "\n" + indent + (String.replicate (max (padLength-1) 0) " ") +  asType Symbol "->" ++ retType
-                if signatureOnly then typeArguments
-                else asType Keyword modifiers ++ functionName ++ asType Symbol ":" + "\n" + typeArguments
+            let allParamsLengths = 
+                many 
+                |> List.map (fun listOfParams ->
+                                 (listOfParams, listOfParams    
```

Comment:
Looks like I am using the `listOfParams` after all [here](https://github.com/nosami/xamarin-monodevelop-fsharp-addin/blob/right-align-tooltips/MonoDevelop.FSharpBinding/FSharpSymbolHelper.fs#L418)

Is there a better way I can do this?